hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHAWithData.java

```diff
@@ -467,20 +467,11 @@ public void testOMRestart() throws Exception {
     // Get the latest snapshotIndex from the leader OM.
     long leaderOMSnaphsotIndex = leaderOM.getRatisSnapshotIndex();
 
```

Comment:
I believe it's better to assert `lastAppliedTxOnFollowerOM < leaderOMSnaphsotIndex` here.

Reply:
And `leaderOMSnaphsotIndex` should be got before `followerOM1.restart()`.